pkg/sources/github/github.go

```diff
@@ -964,13 +978,9 @@ func (s *Source) addOrgsByUser(ctx context.Context, user string) {
 		}
 		logger.V(2).Info("Listed orgs", "page", orgOpts.Page, "last_page", resp.LastPage)
 		for _, org := range orgs {
-			var name string
-			if org.Login != nil {
-				name = *org.Login
-			} else {
+			if org.Login == nil {
 				continue
 			}
-			common.AddStringSliceItem(name, &s.orgs)
 		}
```

Comment:
It looks like this loop essentially does nothing now. Can it be deleted? Or should it be doing something?

Reply:
agreed, I don't think it does anything anymore. NIce find.

Reply:
Oh, wait this means we need an orgCache similar to our repos. I'll get that fixed.